bin/load-test.js

```diff
@@ -0,0 +1,200 @@
+var debug         = require('debug')('queue:bin:load-test');
+var base          = require('taskcluster-base');
+var Promise       = require('promise');
+var _             = require('lodash');
+var v1            = require('../routes/api/v1');
+var https         = require('https');
+var http          = require('http');
+var hawk          = require('hawk');
+var taskcluster   = require('taskcluster-client');
+var assert        = require('assert');
+
+/** Launch server */
+var launch = async function(profile) {
+  debug("Launching with profile: %s", profile);
+
+  // Load configuration
+  var cfg = base.config({
```

Comment:
This load test is neat. Just to make things more DRY, any chance we wrap this base.config({...}) in a utility function? The chance of an env var's name changing may be small; but it would be nice to have one source of truth in any case.